kanidmd_web_ui/src/login.rs

```diff
@@ -405,13 +455,19 @@ impl LoginApp {
     }
 }
 
+unsafe fn console_log(message: String) {
```

Comment:
Why does this need to be unsafe?

Reply:
`rust-analyzer` had a hissy-fit about `gloo::console::log!()` being unsafe, I will revert the changes if it's invalid 😇

Reply:
Yeah, rust-analyzer is being silly. It's fine.

Reply:
Yeah, it's being really silly, there is no unsafe in gloo console at all, so I have nfi what's going on here. Perhaps it's somewhere in the wasm-bindgen bits underneath, but this is NOT unsafe at all from what I can see.

Reply:
I'll revert, won't take long to do 🥳

Reply:
Thanks :)